feat(rfs): add resource to manage a execution plan (#5866)
huaweicloud/terraform-provider-huaweicloud: docs/resources/rfs_execution_plan.md

---
subcategory: "Resource Formation (RFS)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_rfs_execution_plan"
description: |-
  Use this resource to manages a execution plan under specified resource stack within HuaweiCloud.
---

# huaweicloud_rfs_execution_plan

Use this resource to manages a execution plan under specified resource stack within HuaweiCloud.

## Example Usage

```hcl
variable "stack_name" {}
variable "execution_plan_name" {}

> variable template_uri"" {}

Reviewer: ```suggestion
variable "template_uri" {}
```

variable "vars_uri" {}

resource "huaweicloud_rfs_execution_plan" "test" {
  stack_name   = var.stack_name
  name         = var.execution_plan_name
  template_uri = var.template_uri
  vars_uri     = var.vars_uri
}
```

## Argument Reference

The following arguments are supported:

> * `region` - (Optional, String, ForceNew) The region where the private hook is located.

Reviewer: ```suggestion
* `region` - (Optional, String, ForceNew) The region where the related stack to which the execution plan belongs is located.
```

  Changing this creates a new resource.

> * `stack_name` - (Required, String, ForceNew) The name of the resource stack to which the excution plan belongs.

Reviewer: ```suggestion
* `stack_name` - (Required, String, ForceNew) The name of the resource stack to which the execution plan belongs.
```
please check the other places.

  Changing this creates a new resource.
  
* `name` - (Required, String, ForceNew) The name of the execution plan.

>   Change this parameter will create a new resource.  

Reviewer: please consider which describe message is used perfer as follows:
```
Change this parameter will create a new resource. 
```
```
Changing this creates a new resource.
```
